services/core-api/app/api/mines/incidents/resources/mine_incidents.py

```diff
@@ -178,6 +173,21 @@ def post(self, mine_guid):
                     rec_string, mine_incident_id=incident.mine_incident_id)
                 new_recommendation.save()
 
+        categories = data.get('categories')
+        if categories is not None:
+            for category in categories:
+                if category is None:
+                    continue
+                new_category = MineIncidentCategoryXref(
+                    mine_incident_id=incident.mine_incident_id,
+                    mine_incident_category_code=category)
+                incident.categories.append(new_category)
```

Comment:
Follows the pattern used in "mine_incident_doc" above.

Reply:
we shouldn't have to create xref records in this instance. Look at lines 138-144 of this file.